site/compiler/parse-router/index.js

```diff
@@ -0,0 +1,45 @@
+import { flatMap } from '../../../lib/array';
+
+function fullPath(route, prefix) {
+  return (prefix + (route.props.path || '')).replace(/\/+/, '/');
```

Comment:
I find it quite difficult to read this function and understand what is going on. Would you mind adding a comment to explain?